src/components/ApartmentApplicantList/index.js

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react';
+import 'date-fns';
+import { Grid, Card, CardHeader, CardContent, List, Button, Typography } from '@material-ui/core';
+import GroupAddIcon from '@material-ui/icons/GroupAdd';
+import ErrorIcon from '@material-ui/icons/Error';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import { gordonColors } from '../../theme';
+import GordonPeopleSearch from '../Header/components/PeopleSearch';
+import GordonLoader from '../../components/Loader';
+import ApplicantListItem from './components/ApplicantListItem';
+import '../../views/ApartmentApp/apartmentApp.css';
+import '../../views/PeopleSearch/components/PeopleSearchResult/peopleSearchResult.css';
+const MIN_NUM_APPLICANTS = 2;
+const styles = {
+  success: {
+    color: gordonColors.secondary.green,
+    fontSize: '26px',
+  },
+  error: {
+    color: gordonColors.secondary.red,
+    fontSize: '26px',
+  },
+};
+
+// Create a list of applicants, displayed by name, username, and class standing.
+export default class ApplicantList extends Component {
+  constructor(props) {
+    super(props);
+    this.handleSelection = this.handleSelection.bind(this);
+    this.handleRemove = this.handleRemove.bind(this);
+    this.handleSaveButtonClick = this.handleSaveButtonClick.bind(this);
+    this.state = {
+      saveButtonClicked: false,
```

Comment:
That was in the very outdated version of the code, and it was part of one of my earlier attempts to change the look or behavior of the save button during or after saving. That whole feature is handled in a completely different way now, so that state variable is long-gone